AutoPartsShop.WebUI/Pages/DisplaySpinner.razor

```diff
@@ -0,0 +1 @@
+﻿<div class="lds-spinner"><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div></div>
```

Comment:
All these `<div></div>` divs look weird. Are you sure they are all needed considering they are just empty ones?